sdk/tables/azure-data-tables/azure/data/tables/_generated/_azure_table.py

```diff
@@ -23,12 +25,11 @@ class AzureTable(object):
     """AzureTable.
 
     :ivar table: TableOperations operations
-    :vartype table: azure_table.operations.TableOperations
+    :vartype table: azure.table.operations.TableOperations
     :ivar service: ServiceOperations operations
-    :vartype service: azure_table.operations.ServiceOperations
-    :param url: The URL of the service account or table that is the targe of the desired operation.
+    :vartype service: azure.table.operations.ServiceOperations
```

Comment:
this seems like it's missing part of the namespace - shouldn't it be azure.data.tables?

Reply:
Yes it should be